### DISCUSSION TITLE
Feature request: monitor profiles

### DISCUSSION THREAD

**Discussion by etranger** [rxhanson/RectanglePro-Community#84] [Ideas]
I think that using Rectangle Pro in different work scenarios may require different layouts and shortcuts, external monitors being largely different in size compared to built-in displays.

I'd like to have an option to:

1) preferably, be able to create profiles based on connected displays configurations, with custom shortcuts being linked to a specific profile (or work by default for all profiles, as they do today)
2) alternatively, only enable specific custom shortcuts when certain display is connected / active

**Comment by rxhanson** [comment]
Thanks for the feedback! I do have plans to add per display configuration to custom shortcuts in an upcoming release. I don’t have a timeframe for it though.
